lib/osut/utils.rb

```diff
@@ -1146,24 +1148,94 @@ def insulatingLayer(lc = nil)
   # fully-formed ENCLOSED spaces without thermal zoning information or setpoint
   # temperatures (early design stage assessments of form, porosity or
-  # envelope), all OpenStudio spaces will be considered CONDITIONED, presuming
-  # setpoints of ~21°C (heating) and ~24°C (cooling).
+  # envelope), OpenStudio spaces are considered CONDITIONED by default. This
+  # default behaviour may be reset based on the (Space) AdditionalProperties
+  # "space_conditioning_category" key (4x possible values), which is relied
+  # upon by OpenStudio-Standards:
+  #
+  #   github.com/NREL/openstudio-standards/blob/
+  #   d2b5e28928e712cb3f137ab5c1ad6d8889ca02b7/lib/openstudio-standards/
+  #   standards/Standards.Space.rb#L1604C5-L1605C1
+  #
+  # OpenStudio-Standards recognizes 4x possible value strings:
+  #   - "NonResConditioned"
+  #   - "ResConditioned"
+  #   - "Semiheated"
+  #   - "Unconditioned"
+  #
+  # OSut maintains existing "space_conditioning_category" key/value pairs
+  # intact. Based on these, OSut may generate related return values in any of
+  # its methods, e.g. status, postulated setpoints:
+  #
+  #   "space_conditioning_category" | OSut status   | heating °C | cooling °C
+  # -------------------------------   -------------   ----------   ----------
+  #   - "NonResConditioned"           CONDITIONED     21.0         24.0
+  #   - "ResConditioned"              CONDITIONED     21.0         24.0
+  #   - "Semiheated"                  SEMIHEATED      15.0         NA
+  #   - "Unconditioned"               UNCONDITIONED   NA           NA
+  #
+  # OSut also looks up another (Space) AdditionalProperties 'key',
+  # "indirectlyconditioned" to flag plenum or occupied spaces indectly
+  # conditioned with transfer air only. The only accepted 'value' for an
+  # "indirectlyconditioned" 'key' is the name (string) of another (linked)
+  # space, e.g.:
+  #
+  #   "indirectlyconditioned" space | linked space, e.g. "core_space"
+  # -------------------------------   ---------------------------------------
+  #   return air plenum               occupied space below
+  #   supply air plenum               occupied space above
+  #   dead air space (not a plenum)   nearby occupied space
+  #
+  # OSut doesn't validate whether the "indirectlyconditioned" space is actually
+  # adjacent to its linked space. It nonetheless relies on the latter's
+  # conditioning category (e.g. CONDITIONED, SEMIHEATED) to determine
+  # anticipated ambient temperatures in the former. For instance, an
+  # "indirectlyconditioned"-tagged return air plenum linked to a SEMIHEATED
+  # space is considered as free-floating in terms of cooling, and unlikely to
+  # have ambient conditions below 15°C under heating (winter) design
+  # conditions. OSut will associate this plenum to a 15°C heating setpoint
+  # temperature. If the SEMIHEATED space instead has a heating setpoint
+  # temperature of 7°C, then OSut will associate a 7°C heating setpoint to this
+  # plenum.
```

Comment:
Revisiting how _unconditioned_ (e.g. attics) vs _indirectly-conditioned_ (e.g. plenums) spaces are caught and treated. This had become quite a challenge when processing 3rd-party models (i.e. not US DOE Prototypes).

In _OpenStudio_, plenums fit very neatly with ASHRAE 90.1 definitions. In _OpenStudio-Standards_, plenums as a concept is somewhat _looser_, often simply shorthand for _indirectly-conditioned_ spaces. The inline comments go into some detail as to why this can be an issue for 3rd-party models. This revised treatment remains as close to both OpenStudio & OpenStudio-Standards approaches as possible, yet only to an extent with the latter.

When it comes to determining which surfaces are to be considered as part of the building envelope, a space's status as a plenum is largely irrelevant. Relying on a combo of (new) _unconditioned?_ and SDK's _partofTotalFloorArea_ is simpler, safer and sufficient.